services/QuillLMS/app/models/districts_admins.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: districts_admins
+#
+#  id          :bigint           not null, primary key
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  district_id :bigint           not null
+#  user_id     :bigint           not null
+#
+# Indexes
+#
+#  index_districts_admins_on_district_id  (district_id)
+#  index_districts_admins_on_user_id      (user_id)
+#
+class DistrictsAdmins < ApplicationRecord
```

Comment:
While the underlying table name is plural district_admins, the convention for Rails model naming is singular so this would be `DistrictsAdmin` (I think of the model name as being an individual resource for the table). The reason this is important is that there are various inflections that rails does when dealing with resources which assume a singular model name.

Reply:
While we already have `SchoolsAdmins` and `StudentsClassrooms`, there is `ClassroomsTeacher`.